packages/grafana-runtime/src/config.ts

```diff
@@ -27,6 +28,7 @@ export class GrafanaBootConfig implements GrafanaConfig {
   isPublicDashboardView: boolean;
   datasources: { [str: string]: DataSourceInstanceSettings } = {};
   panels: { [key: string]: PanelPluginMeta } = {};
+  auth: AuthSettings = {} as AuthSettings;
```

Comment:
is this `as AuthSettings;` required? it seems empty object is a valid shape for `AuthSettings` anyway?

Reply:
:+1: